device/src/storage.rs

```diff
@@ -1,73 +1,122 @@
 extern crate alloc;
-use crate::state::FrostState;
-use alloc::format;
+use alloc::{format, string::String};
 use bincode::{
     de::read::Reader,
     enc::write::Writer,
     error::{DecodeError, EncodeError},
 };
 use embedded_storage::{ReadStorage, Storage};
 use esp_storage::{FlashStorage, FlashStorageError};
+use frostsnap_core::{message::DeviceToStorageMessage, schnorr_fun::fun::Scalar};
 
-pub const NVS_PARTITION_START: u32 = 0x9000;
-pub const NVS_PARTITION_SIZE: usize = 0x6000;
+const NVS_PARTITION_START: u32 = 0x9000;
```

Comment:
I think you asked @musdom this elsewhere, could we add a comment where these come from?

Reply:
https://docs.espressif.com/projects/esp-idf/en/latest/esp32c3/api-guides/partition-tables.html